ApplicationLibCode/ProjectDataModel/Summary/RimSummaryAddressModifier.cpp

```diff
@@ -30,28 +30,32 @@
 //--------------------------------------------------------------------------------------------------
 ///
 //--------------------------------------------------------------------------------------------------
-RifEclipseSummaryAddress RimSummaryAddressModifier::replaceObjectName( const RifEclipseSummaryAddress&                    sourceAdr,
-                                                                       std::string                                        objectName,
-                                                                       RimSummaryAddressCollection::CollectionContentType contentType )
+RifEclipseSummaryAddress RimSummaryAddressModifier::replaceTokenForCategory( const RifEclipseSummaryAddress&                  sourceAdr,
+                                                                             std::string                                      token,
```

Comment:
token can be `const std::string&` (to avoid copy)?

Reply:
fixed